pkg/asset/installconfig/aws/client.go

```diff
@@ -0,0 +1,144 @@
+package aws
+
+import (
+	"context"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockgen -source=./client.go -destination=.mock/awsclient_generated.go -package=mock
```

Comment:
```suggestion
//go:generate mockgen -source=./client.go -destination=./mock/awsclient_generated.go -package=mock
```

Reply:
I've removed the mock framework from AWS.